src/patternfly/components/Table/table.scss

```diff
@@ -464,6 +511,125 @@
       border-left-width: 0;
     }
   }
+
+  // tr hoverable
+  tr.pf-m-hoverable {
+    cursor: pointer;
+    background-color: var(--pf-c-table--tr--m-hoverable--BackgroundColor);
```

Comment:
could this just be white? with a comment to make it the default for `tr` in a breaking change? Or is there a reason we only set it on hover/focus/active?